### PR TITLE
Patch management: plugin -> core

### PR DESCRIPTION
**Pull request by hius07**
Discussed in https://github.com/koreader/koreader/issues/12841.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/12862)

### CODE REVIEW

**Comment by hius07** [comment]
Sure, thanks!

File: frontend/ui/data/onetime_migration.lua [hunk 1]
```diff
@@ -760,5 +760,13 @@ if last_migration_date < 20241123 then
     end
 end
 
+-- 20241207, Move PatchManagement plugin to the core
+-- https://github.com/koreader/koreader/pull/12862
+if last_migration_date < 20241207 then
+    logger.info("Performing one-time migration for 20241207")
+
+    ffiUtil.purgeDir(DataStorage:getDataDir() .. "/plugins/patchmanagement.koplugin")
```
Comment (poire-z) [inline frontend/ui/data/onetime_migration.lua:768]: The wording of the comment should probably read as:
`-- We moved patch management to core. Remove the original plugin.`

(This migration step does _not_ "Move PatchManagement plugin to the core", _you_ did :))

**Comment by poire-z** [comment]
Wondering:
#12809 did not remove via onetime_migration the plugins it has disabled.
This PR does.
For consistency, would/should there be a preferred method ?

**Comment by hius07** [comment]
https://github.com/koreader/koreader/blob/157c03c42dda85ac3c114762e6beaa5601936243/frontend/pluginloader.lua#L8-L10
They will not be loaded in any case, I can add purging them here. @pazos ?

**Comment by pazos** [comment]
I'm not an expert in migrations.

All the removed plugins until now we're not actively removed, just blacklisted. I'm fine with this approach but if you think it worths to remove them we can do delete them all and remove the obsolete array

**Comment by Frenzie** [comment]
Some of them might have data that's important for a move to contrib. (Emphasis on might, if they don't they can probably be deleted.)

**Comment by hius07** [comment]
The OBSOLETE plugins are not just disabled, they are not visible in the Plugin management and cannot be enabled.

**Comment by hius07** [comment]
Purging Patch management only.

**Pull request merged**